README.md

```diff
@@ -17,6 +17,26 @@ Download this repository keeping the `.tmbundle` extension.
     ``git clone https://github.com/cylc/Cylc.tmbundle <path to sublime packages>/Cylc.tmbundle``
 
    (to get package path click Preferences => Browse Packages)
+   
+## FAQ - How do I enable syntax highlighting for all `.rc` files?
+
```

Comment:
For context would be good to add something along the lines of:

By default this bundle will associate `suite.rc` and `.cylc` files with the Cylc language, however, if you are working with Cylc files which don't match these names...

Reply:
Added

Reply:
Thanks.